CONTRIBUTING.template.md

```diff
@@ -0,0 +1,55 @@
+# [PROJECT] Contribution and Governance Policies
+
+This document describes the contribution process and governance policies of the FINOS [PROJECT] project. The project is also governed by the [Linux Foundation Antitrust Policy](https://www.linuxfoundation.org/antitrust-policy/), and the FINOS [IP Policy](IP-Policy.pdf), [Code of Conduct](Code-of-Conduct.md), [Collaborative Principles](Collaborative-Principles.md), and [Meeting Procedures](Meeting-Procedures.md).
```

Comment:
Could you please use `{project name}` instead of `[PROJECT]` ?

Reply:
Done 👍